test/templates/layout.njk

```diff
@@ -7,6 +7,8 @@
     <title>{{ id }}</title>
     <link rel="stylesheet" href="static/layout.css">
     <style>
+        {{ 'fonts.scss' | component_css }}
```

Comment:
@greg-a-smith should I load the fonts and icons here like you did in the example you showed me with the variables instead?

Reply:
I think it will need to be like this for now because these are `*.scss` files and need to be converted to CSS first.  The other hard-coded `<link>` tags are just loading files that are already CSS.  If we ever refactor the playground to use generated CSS files, then it could just be dropped in as a `<link>` tag.